insecure/corruptlibp2p/fixtures.go

```diff
@@ -0,0 +1,54 @@
+package corruptlibp2p
+
+import (
+	pubsubpb "github.com/libp2p/go-libp2p-pubsub/pb"
+
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+type GossipSubCtrlOption func(*pubsubpb.ControlMessage)
+
+// GossipSubCtrlFixture returns a ControlMessage with the given options.
+func GossipSubCtrlFixture(opts ...GossipSubCtrlOption) *pubsubpb.ControlMessage {
+	msg := &pubsubpb.ControlMessage{}
+	for _, opt := range opts {
+		opt(msg)
+	}
+	return msg
+}
+
+// WithIHave adds iHave control messages of the given size and number to the control message.
+func WithIHave(msgCount int, msgSize int) GossipSubCtrlOption {
+	return func(msg *pubsubpb.ControlMessage) {
+		iHaves := make([]*pubsubpb.ControlIHave, msgCount)
+		for i := 0; i < msgCount; i++ {
+			topicId := gossipSubTopicIdFixture()
+			iHaves[i] = &pubsubpb.ControlIHave{
+				TopicID:    &topicId,
+				MessageIDs: gossipSubMessageIdsFixture(msgSize),
+			}
+		}
+		msg.Ihave = iHaves
+	}
+}
+
+// gossipSubMessageIdFixture returns a random gossipSub message ID.
+func gossipSubMessageIdFixture() string {
+	// TODO: messageID length should be a parameter.
+	return unittest.GenerateRandomStringWithLen(10)
```

Comment:
Move 10 to a const variable.

Reply:
https://github.com/onflow/flow-go/pull/3585/commits/dc80931f84d4cfa3fb7cf770d0fb2a481f008df8